main.js

```diff
@@ -11,8 +11,13 @@ module.exports = (glob, options) => {
   // Main file poses a special case, as its changes are
   // only effective when the process is restarted (hard reset)
   let appPath = app.getAppPath()
-  let config = require(path.join(appPath, 'package.json'))
-  let mainFile = path.join(appPath, config.main)
```

Comment:
I'd just add it in this line:

```js
let mainFile = path.join(appPath, config.main) || 'index.js'
```

what do you reckon?

Reply:
I believe you might have meant:
```
    let mainFile = path.join(appPath, config.main || 'index.js')
```
since this retains `appPath`